### ISSUE TITLE
validation and test set details for convai2

### ISSUE THREAD

**Issue by Henry-E** [facebookresearch/ParlAI#1004]
I'm curious about the breakdown of revised and original personas for the validation and test set.

"[setting aside 200 never seen before personas for validation](http://convai.io/#personachat-convai2-dataset)". Are there a corresponding 200 revised personas also used in validation set? As I understand it both the original and revised persona are used as context for the same dialogue in the training set. So for evaluation is each dialogue seen twice, once with the original persona and once with the revised? And are these numbers roughly equivalent for the test set.

In a more general question I'm wondering is a 50:50 split enough to emphasise the desired regularizing effect of the revised personas on the models. There's two ways I'm thinking this might be an issue.

1. A model which copies heavily from the original persona could do very well, enough to offset the drop in performance when testing on the revised persona. For example this chat I had with one of the participant models (it typed the second utterance too quickly to be human but I could be wrong)
![screen shot 2018-07-21 at 10 19 08](https://user-images.githubusercontent.com/12613144/43030085-c1b477b2-8ccf-11e8-9c61-f0c5442f5663.png)

2. There was an interesting paper at the ACL translation and generation workshop; [On the Impact of Various Types of Noise on Neural Machine Translation](https://arxiv.org/abs/1805.12282). The main takeaway from their study was that providing noisy data that teaches the model to copy the input severely impacts its ability to learn the true task. A model would start decoding normally, then if it started copying a source token this derailed it completely and it continue to copy for the rest of the utterance. The parallel here is that there's a strong incentive for the model to learn to copy much of the source input and having an equal number of revised persona examples might not be enough to balance it out. Especially if the desired goal is to have the personas guide conversation in a more abstract or contextual way rather than taking advantage of word overlap as mentioned in the paper. 
![screen shot 2018-07-21 at 09 43 48](https://user-images.githubusercontent.com/12613144/43030195-bd181f8c-8cd0-11e8-81ba-06668ec0aa77.png)
